src/client/pythonEnvironments/common/environmentManagers/condaService.ts

```diff
@@ -19,6 +19,42 @@ export class CondaService implements ICondaService {
         @inject(IFileSystem) private fileSystem: IFileSystem,
     ) {}
 
+    public async getActivationScriptFromInterpreter(
+        interpreterPath?: string,
+        envName?: string,
+    ): Promise<string | undefined> {
+        const condaPath = await this.getCondaFileFromInterpreter(interpreterPath, envName);
+        if (condaPath) {
+            const activatePath = path.join(path.dirname(condaPath), 'activate').fileToCommandArgumentForPythonExt();
+            if (await this.fileSystem.fileExists(activatePath)) {
+                return activatePath;
+            }
+        }
+
+        if (this.platform.isLinux || this.platform.isMac) {
+            const condaInfo = await this.getCondaInfo();
+            // eslint-disable-next-line camelcase
+            if (condaInfo?.root_prefix) {
+                return (
+                    path
+                        // eslint-disable-next-line camelcase
+                        .join(condaInfo.root_prefix, 'bin', 'activate')
+                        .fileToCommandArgumentForPythonExt()
+                );
+            }
+        }
+
+        return undefined;
```

Comment:
```suggestion
        return 'activate';
```
In case we're not able to find the full path to activate script, just use `'activate'` and hope it's on user's `PATH`.